frontend/src/customHooks/useUser.ts

```diff
@@ -46,9 +47,22 @@ export default function useUser(loadAllGames: () => void) {
                 setUser(undefined);
             })
             .catch(() => {
-                toast.error("Logout hat nicht funktioniert:");
+                console.log("error")
```

Comment:
why do you log this to console? Everywhere else in this component you use the toast.error

Reply:
because my Toast wont be Displayed at the LoginPage and i dunno why. when i add ToastContainer nothing happens